.github/workflows/publish-chart.yaml

```diff
@@ -18,7 +23,9 @@ env:
   KIND_VERSION: "v0.14.0"
   KIND_IMAGE: "kindest/node:v1.23.6@sha256:b1fa224cc6c7ff32455e0b1fd9cbfd3d3bc87ecaa8fcb06961ed1afb3db0f9ae"
 jobs:
+  # `test-chart` job starts if a PR with Helm Chart is created, merged etc.
   test-chart:
+    if: github.event_name != 'push'
     runs-on: ubuntu-20.04
```

Comment:
ubuntu-24.04 is currently `latest` version of `ubuntu` runner.
Can we bump runner version (maybe in another PR)

Reply:
done https://github.com/aquasecurity/trivy/pull/7526/commits/649987e42e78d12af52eacfa65272e8b73fa64c7
I've update it here, because another actions already are bumped os version up (or using `-latest`).